dockerHashing/src/main/scala/cromwell/docker/DockerRegistryConfig.scala

```diff
@@ -1,22 +1,21 @@
 package cromwell.docker
 
 import java.util.concurrent.Executors
-
 import cats.syntax.apply._
 import common.validation.Validation._
 import net.ceedubs.ficus.Ficus._
 import com.typesafe.config.Config
 import common.validation.ErrorOr.ErrorOr
 import cromwell.core.io.Throttle
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor}
 
 case class DockerRegistryConfig(throttle: Option[Throttle], nbThreads: Int) {
-  lazy val executionContext = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(nbThreads))
+  lazy val executionContext: ExecutionContextExecutor = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(nbThreads))
 }
 
 object DockerRegistryConfig {
-  lazy val default = DockerRegistryConfig(None, 5)
+  lazy val default: DockerRegistryConfig = DockerRegistryConfig(None, 5)
```

Comment:
ToL: I like cleanup but personally hold back to only do it when there are other material changes in the file I'm already editing. Out of curiosity were any of these changes in this file required for the PR to pass?